cmp/grid/GridModel.js

```diff
@@ -172,6 +172,8 @@ export class GridModel extends HoistModel {
 
     /**
      * @param {Object} c - GridModel configuration.
+     * @param {int} [c.clicksToEdit] -  Set how many clicks are needed to open an editable cell for edting. 
```

Comment:
Type here would be `{number}`

Reply:
Ok.  I saw {int} in these 2 places, so thought it was accepted:
https://github.com/xh/hoist-react/blob/develop/utils/js/LangUtils.js#L210
https://github.com/xh/toolbox/blob/develop/client-app/src/core/svc/PortfolioService.js#L21

JSDoc wouldn't choke on "int", but are we trying to stick to real javascript types?

Reply:
Yes - jsDoc will let you put literally anything in there, but if it doesn't resolve to a known JS primitive or another "namepath" that jsdoc can resolve, I don't think it will have any (meaningful) effect.

I'll clean up those other two references, thanks for catching. 👍